admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+import Router from '../../../components/router';
+import OrderViewPageMap from '../OrderViewPageMap';
+import {EventEmitter} from '../../../components/event-emitter';
+import OrderViewEventMap from './order-view-event-map';
+
+const $ = window.$;
+
+export default class OrderProductEdit {
+  constructor(orderProductId) {
+    this.router = new Router();
+    this.orderProductId = orderProductId;
+    this.productRow = $(`#orderProduct_${this.orderProductId}`);
+    this.productRowEdit;
+    this.productEditActionBtn = $(OrderViewPageMap.productEditActionBtn);
+    this.productIdInput = $(OrderViewPageMap.productEditOrderDetailInput);
+    this.priceTaxIncludedInput = $(OrderViewPageMap.productEditPriceTaxInclInput);
+    this.priceTaxExcludedInput = $(OrderViewPageMap.productEditPriceTaxExclInput);
+    this.taxRateInput = $(OrderViewPageMap.productEditTaxRateInput);
+    this.quantityInput = $(OrderViewPageMap.productEditQuantityInput);
+    this.product = {};
+  }
+
+  setupListener() {
+    this.productRowEdit.find(OrderViewPageMap.productEditPriceTaxInclInput).on('change keyup', (event) => {
+      let priceTaxIncl = parseFloat(event.target.value);
+      if (priceTaxIncl < 0 || isNaN(priceTaxIncl)) {
+        priceTaxIncl = 0;
+      }
+      const taxRate = this.productRowEdit.find(OrderViewPageMap.productEditTaxRateInput).val() / 100 + 1;
+      this.productRowEdit.find(OrderViewPageMap.productEditPriceTaxExclInput).val(ps_round(priceTaxIncl / taxRate, 2));
+    });
+    this.productRowEdit.find(OrderViewPageMap.productEditPriceTaxExclInput).on('change keyup', (event) => {
+      let priceTaxExcl = parseFloat(event.target.value);
```

Comment:
Ah, here we could reuse it 💪 the JS tax calculations manager

Reply:
:ballot_box_with_check: Thanks